File: University/TarikheTavallodS1.go
package main
import "strings"
import "os"
import "fmt"

func get_input[T any]() T {
	var i T
	fmt.Scan(&i)
	return i
}
type Main struct {
}
func Main_1() * Main {
    var this * Main =  &Main{};
    return this;
}
func (this *Main) main_1(args []string) {
    try () {
        var  number  string = get_input[string]();
        fmt.Println("saal:" + string(number[0]) + string(number[1]));
        fmt.Println("maah:" + string(number[2]) + string(number[3]));
    }
}

func main(){
	var task = Main{};
	task.main_1(os.Args);
}

File: University/TarikheTavallodS1.php
<?php

class Main
{
    function __construct(){
    }
    public static function Main_1()
    {
        $local_this = new Main();
        return $local_this;
    }
    public static function main_1(&$args)
    {
        try (
        public $scanner = "Inputs";)
        {
            $number = readline();
            echo "saal:" . strval($number[0]) . strval($number[1]),"\n";
            echo "maah:" . strval($number[2]) . strval($number[3]),"\n";
        }
    }
}
Main::main_1($argv);
